site/static/css/docs.css

```diff
@@ -5,7 +5,7 @@ main {
 
 @media (max-width: 350px) {
   main {
-    width: 80vw;
+    width: 90vw;
```

Comment:
Not sure why we need to make it wider?

Reply:
I think the whitespaces around was taking too much space. Kinda squeezing the code snippet
<img src="https://user-images.githubusercontent.com/33410545/91606966-c05b6900-e990-11ea-92db-1836187fb7be.png" height="600px">.

This much width makes it look much more readable imo . Don't you think ?
<img src="https://user-images.githubusercontent.com/33410545/91607459-8a6ab480-e991-11ea-99bc-38f2ae1381fa.png" height="600px">

Reply:
hm yeah this makes sense